src/js/_enqueues/vendor/plupload/handlers.js

```diff
@@ -613,6 +613,11 @@ jQuery( document ).ready( function( $ ) {
 					wpQueueError( pluploadL10n.noneditable_image );
 					up.removeFile( file );
 					return;
+				} else if ( file.type === 'image/heic' && up.settings.heic_upload_error ) {
```

Comment:
This condition already exists above on line 603.

Reply:
good point, I will leave the current behavior and remove this bit